content/engineering-practices-fishbowl-dev-cop.md

```diff
@@ -0,0 +1,42 @@
+Title: Fishbowl Dev CoP
+Date: 2023-12-14
+Category: Posts 
+Tags: engineering, technical-excellence, dev-cop, fishbowl
+Slug: engineering-practices-fishbowl-dev-cop
+Author: Martin M. Lacey
+Summary: Experimenting with a New Developer CoP Format: Fishbowl Discussions
+
+In this week's podcast, we're diving into a fresh approach for our Developer Community of Practice (Dev CoP). Over the years, we’ve leaned on standard presentations and open dialogue formats, but this time, we’re shaking things up with a fishbowl discussion. As part of our Scalable Agile Framework for the Enterprise (SAFe) practices at WorkSafeBC, this experiment aligns with our commitment to fostering innovation and engagement in our CoP ceremonies.
+
+# What Are Communities of Practice?
+
+Communities of Practice (CoPs) are groups of professionals united by a shared interest in a technical or business domain. They collaborate to exchange knowledge, refine their skills, and push the boundaries of their field. This concept, outlined by Etienne Wenger in Communities of Practice: Learning, Meaning, and Identity, emphasizes learning through collaboration and shared experiences.
+
+For more, see: Communities of Practice - Scaled Agile Framework.
```

Comment:
... and https://wsbctechnicalblog.github.io/ceremony-community-of-practice.html? Self-advertising :)